FlowCrypt.xcodeproj/xcshareddata/xcschemes/FlowCrypt.xcscheme

```diff
@@ -7,7 +7,7 @@
       buildImplicitDependencies = "YES">
       <BuildActionEntries>
          <BuildActionEntry
-            buildForTesting = "NO"
+            buildForTesting = "YES"
```

Comment:
Please revert this change. If you need a separate debug build, please set up some unrelated build target that doesn't affect prod builds.

Reply:
Let me check tomorrow why this changes occurred. I have just opened a scheme to see what arguments it has. 
Will get back.

Reply:
Got it. Thanks

Reply:
From what I found this option should be set to "yes" in our case. 
If this option turned on, xcodebuild (command line tool) will take workspace scheme and create ‘XCTESTRUN’ file, which allows to build app for testing and create 'xctestrun' file. This allows to run tests on CI. 

So in case you open edit scheme and close it this changes occurs
<img width="931" alt="Screenshot 2021-07-09 at 14 22 02" src="https://user-images.githubusercontent.com/13693255/125071047-73bce000-e0c1-11eb-9224-a2ecda7cb8a8.png">